graylog2-web-interface/src/views/components/dashboard/BigDisplayModeConfiguration.test.jsx

```diff
@@ -11,6 +11,7 @@ import Query from 'views/logic/queries/Query';
 import ViewState from 'views/logic/views/ViewState';
 import type { ViewStateMap } from 'views/logic/views/View';
 import BigDisplayModeConfiguration from './BigDisplayModeConfiguration';
+import { asElement } from '../../../../test/wrappedTestingLibrary';
```

Comment:
This import could be shortened to

```suggestion
import { asElement } from 'wrappedTestingLibrary';
```